docs/Basics-Layout.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: basics-layout
-title: Layout
+title: Layout with Flexbox
 layout: docs
 category: The Basics
-permalink: docs/basics-layout.html
+permalink: docs/flexbox.html
```

Comment:
Google. This page is one of the most visited on the site right now. People search for [react native flexbox] and it's the first link.